<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(15, 125, 209);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>491</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 26pt &quot;MS Shell Dlg 2&quot;;color: rgb(255, 255, 255);
</string>
    </property>
    <property name="text">
     <string>PUNTO DE VENTA EDUSISTEM</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Código Cliente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigoCliente">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Código Producto:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCodigoProducto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Nombres y Apellidos:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDescripcion">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>180</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Descripción:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Stock:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtStock">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>140</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Precio:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPrecio">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCantidad">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Cantidad:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>FECHA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFecha">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>N°</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNumeroDocumento">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>VENTAS DE PRODUCTOS TECNOLÓGICOS</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tblDetalle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>751</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <column>
     <property name="text">
      <string>Nro</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Código de Producto</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Descripción</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Precio</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Cantidad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Total</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 38, 9);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>AGREGAR</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../IMG/registrar.png</normaloff>../IMG/registrar.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>SUB TOTAL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSubtotal">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>440</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtIgv">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>480</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTotal">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>480</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>  IGV (18%):</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>520</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>TOTAL A PAGAR:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCenerar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 38, 9);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>GENERAR VENTA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnImprimir">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>450</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 38, 9);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>IMPRIMIR FACTURA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCerrar">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>450</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(200, 38, 9);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CERRAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscarCliente">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscarProducto">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>180</y>
      <width>16</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
